python/code_challenges/tree_fizz_buzz.py

```diff
@@ -1,5 +1,26 @@
-from data_structures.binary_tree import BinaryTree
+from data_structures.kary_tree import KaryTree
+from data_structures.queue import Queue
 
+def fizz_buzz_tree(tree):
+    q = Queue()
+    new_tree = KaryTree(tree.root)
 
-def fizz_buzz_tree():
-    pass
+    q.enqueue(new_tree.root)
+    while not q.is_empty():
+        front = q.dequeue()
+        front.value = function(front.value)
+        for i in front.children:
+            q.enqueue(i)
+
+    return new_tree
+
+
+def function(num):
```

Comment:
This function name makes it hard for someone else to determine it's meaning. The name should indicate what the function does.